scss/main.scss

```diff
@@ -356,13 +356,23 @@ h1,h2,h3 {
 
 .contact-form-container {
   color: $white;
-  padding-bottom: 5rem;
-  margin-bottom: 2rem;
+  padding: 80px 96px;
   background-color: $cfa-red;
+
+  @media(max-width: 768px){
+    padding: 64px 48px;
+  }
+
+  @media(max-width: 360px){
+    padding: 48px 24px;
+  }
+
   h1 {
     color: $white;
-    margin-bottom: 2rem;
     font-weight: 700;
+    @media(max-width: 651px){
+      font-size: 32px;
```

Comment:
Let's please use `rem` instead of `px`.
This will allow users to scale fonts up if they need to.

Reply:
Thanks for the tip! It should be fixed now.